src/profiler/profilerthread.cpp

```diff
@@ -126,7 +126,7 @@ void ProfilerThread::sample(const SAMPLE_TYPE timeSpent)
 		}
 	}
 
-	for (ptrdiff_t n = (ptrdiff_t)count; n >= 0; --n)
+	for (ptrdiff_t n = (ptrdiff_t)count-1; n >= 0; --n)

```

Comment:
Please squash this change into the commit that added this code.

Reply:
I'm not sure what you mean.
The original commit (by someone else) where this bug was introduce?

Reply:
Ah, my bad; I thought this code was added in an earlier commit in this pull request.

Reply:
It would be great if the commit message for this change explained the fix and mention the commit where this bug was introduced.

Reply:
I tried blame, but apparently it looked like it has been around ever since the code was moved from C# to C++.

Reply:
I must have been dreaming.